tests/test_pandas_dataset.py

```diff
@@ -107,116 +107,6 @@ def test_expect_column_values_to_be_unique(self):
-
-        D2 = ge.dataset.PandasDataset({
-            'a' : [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
-            'b' : [1, 2, 3, 4, 5, 6, 7, 8, 9, None],
-        })
-        D2.set_default_expectation_argument("result_format", "COMPLETE")
-
-        #assert_equal(
-        #    D.expect_column_values_to_not_be_null('x'),
-        #    {'success':False, 'unexpected_list':[None]}
-        #)
-
-        T = [
-                {
-                    'in':['a'],
-                    'kwargs':{},
-                    'out':{'success':True, 'unexpected_index_list':[], 'unexpected_list':[]}},
-                {
-                    'in':['a'],
-                    'kwargs':{'mostly':.90},
-                    'out':{'success':True, 'unexpected_index_list':[], 'unexpected_list':[]}},
-                {
-                    'in':['b'],
-                    'kwargs':{},
-                    'out':{'success':False, 'unexpected_index_list':[9], 'unexpected_list':[None]}},
-                {
-                    'in':['b'],
-                    'kwargs':{'mostly':.95},
-                    'out':{'success':False, 'unexpected_index_list':[9], 'unexpected_list':[None]}},
-                {
-                    'in':['b'],
-                    'kwargs':{'mostly':.90},
-                    'out':{'success':True, 'unexpected_index_list':[9], 'unexpected_list':[None]}}
-        ]
-
-        for t in T:
-            out = D2.expect_column_values_to_not_be_null(*t['in'], **t['kwargs'])
-            self.assertEqual(t['out']['success'], out['success'])
-            self.assertEqual(t['out']['unexpected_index_list'], out['result']['unexpected_index_list'])
-            self.assertEqual(t['out']['unexpected_list'], out['result']['unexpected_list'])
-
-        D3 = ge.dataset.PandasDataset({
-            'a' : [None, None, None, None],
-        })
-        D3.set_default_expectation_argument("result_format", "COMPLETE")
-
-        T = [
-                {
-                    'in':['a'],
-                    'kwargs':{},
-                    'out':{'success':False, 'unexpected_index_list':[0,1,2,3], 'unexpected_list':[None,None,None,None]}
-                },
-                {
-                    'in':['a'],
-                    'kwargs':{"mostly":.95},
-                    'out':{'success':False, 'unexpected_index_list':[0,1,2,3], 'unexpected_list':[None,None,None,None]}
-                },
-        ]
-
-        for t in T:
-            out = D3.expect_column_values_to_not_be_null(*t['in'], **t['kwargs'])
-            # out = D3.expect_column_values_to_be_null(*t['in'], **t['kwargs'])
-            self.assertEqual(t['out']['success'], out['success'])
-            self.assertEqual(t['out']['unexpected_index_list'], out['result']['unexpected_index_list'])
-            self.assertEqual(t['out']['unexpected_list'], out['result']['unexpected_list'])
-
     def test_expect_column_values_to_be_null(self):
```

Comment:
I've been working on this for awhile now, y'all should take a look at it @jpcampb2 @abegong.

- SQLAlchemy doesn't recognize `NULL` values using the `in_` clause, you will see updated logic for that. 
- Python native `in` also doesn't recognize `np.nan`, logic for mapping the ignored/null values updated in the column map expectation decorator
- Made a `util` helper function for `np.nan` equality testing, because those are annoying.
- Migrated the `expect_column_values_to_be_null` tests


It seems that for two tests instead of returning the unexpected value in a list (`[None]`) it is returning it a single value (`None`).  I wonder if this has something to do with `np.nan`...